### ISSUE TITLE
Enhancements

### ISSUE THREAD

**Issue by DaveCasson** [CH-Earth/summaWorkflow_public#52]
1. - The Jupyter Notebooks are all look for a kernel as follows: Could not find a kernel matching Python [conda env:geospatialTools_candexTest]. Setting the kernels to summa-venv (or summa-conda-env) would avoid this.

2.  Create small utility functions to hold commonly used commands such as:
-read_from_control( file, setting )
- make_default_path(suffix):
- Create a log file, suggest using logger package

**Comment by wknoben** [comment]
Thanks, I'll take these under consideration.

**Comment by wknoben** [comment]
Leaving this here for future reference (scripts only): https://gis.stackexchange.com/questions/177882/getting-script-file-name-as-object-in-python

**Comment by wknoben** [comment]
Fixed the first one in #55.

**Comment by tgq14** [comment]
I agree with Dave as I also planned to suggest this. I think you can also create some common functions for shell scripts and import them when you need to use them. For example, many files have a recent commit of "Updated bash scripts to ignore lines starting with "#".". It is tiring to update files one by one and you may miss some scripts sometimes. Future adjustments are also possible.
